definitions/private-law/json/AuthorisationCaseEvent/Gatekeeper/FL401/AuthorisationCaseEvent.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "LiveFrom": "04/11/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseEventID": "fl401SendToGateKeeper",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-privatelaw-courtadmin",
+          "caseworker-privatelaw-judge",
+          "caseworker-privatelaw-la",
+          "caseworker-privatelaw-systemupdate"
+        ],
+        "CRUD": "CRU"
```

Comment:
We need solicitor and other user read access, so that they can visit the history